website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * New and Noteworthy Projects
+ */
+
+var $ = require('jquery');
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+
+// CSS
+require('css/quick-project-search-plugin.css');
+
+// XHR config for apiserver connection
+var xhrconfig = function(xhr) {
+    xhr.withCredentials = true;
+};
+
+
+var newAndNoteworthy = {
+    controller: function() {
+        var self = this;
+        self.newAndNoteworthyNodes = m.prop([]);
+        self.popularNodes = m.prop([]);
+        self.contributorsMapping = {};
+
+        // Load new and noteworthy nodes
+        var newAndNoteworthyUrl = $osf.apiV2Url('nodes/' + window.contextVars.newAndNoteworthy + '/node_links/', {});
+        var newAndNoteworthyPromise = m.request({method: 'GET', url: newAndNoteworthyUrl, config: xhrconfig});
+        newAndNoteworthyPromise.then(function(result){
+            var numPopular = Math.min(result.data.length - 1, 4);
```

Comment:
It looks like you are setting 5 as the total shown. It's best to make this a constant variable 
```js
var SHOW_TOTAL = 5;
```

also on ~L30 don't use ```1 <= numPopular``` but adjust your code so you use ```1< numPopular```

so I guess something like this
```js
var SHOW_TOTAL = 5;

var l;
var numPopular = Math.min(result.data.length, SHOW_TOTAL);
for (l = 0; l < numPopular; l++){
```